bin/get_metadata.py

```diff
@@ -82,6 +84,14 @@ def get_bcl2fastq_version(self, runfolder):
             bcl2fastq_str = f.read()
         return bcl2fastq_str.split("v")[1].strip()
 
+    def get_software_version(self, runfolder):
+        with open(Path(runfolder) / "pipeline_info" / "software_versions.yml") as f:
+            return {
+                software: version
+                for sotfware_dict in yaml.safe_load(f).values()
+                for software, version in sotfware_dict.items()
```

Comment:
Super minor: Typo in variable name `sotfware_dict` ? Might be nice to fix.

Reply:
Oh I see, the typo was for all the instances of this variables :sweat_smile: